packages/widget/src/stores/form/FormStore.tsx

```diff
@@ -130,6 +146,14 @@ export const FormStoreProvider: React.FC<FormStoreProviderProps> = ({
         hiddenToAddress,
       ),
     );
+
+    // emit sendToWalletAddressChange event if toAddress is set in the config
+    if (toAddress) {
+      // debounce used to push this event emit to the back of the job queue
+      // otherwise without it the useEffect that is set up for listening to widget events
+      // can potentially be set up after the event has been emitted
+      debouncedToAddressWidgetEvent(toAddress.address, widgetEvents);
+    }
```

Comment:
Do you think we need to emit an event during initialization? I'm not a fan of debouncing logic here.

Reply:
I'm happy to remove it if you don't think its needed

Reply:
yes, please 🙂

Reply:
Have remove this